### ISSUE TITLE
In Firefox Developer Edition, all scripts and settings gone

### ISSUE THREAD

**Issue by joeytwiddle** [Tampermonkey/tampermonkey#967]
## Problem

In Firefox, all of my scripts have disappeared, settings have reset, and nothing is restoring from the Google Drive sync.

## Debugging

There are two things which might be causing this:

1. I am using Firefox Developer Edition. Perhaps they have changed an API and broke TamperMonkey.

2. I use Google Drive to sync settings between this Firefox and Chrome browser.
   In Chrome, I recently enabled the checkbox **"Include Tampermonkey settings"**
   (I was hoping to send my editor settings to Firefox)
   Perhaps for some reason, this has caused a problem for Tampermonkey in Firefox.

Also worth noting:

- In Firefox, I did **"Remove Access Token"** for Google Drive

  I rebooted Firefox, but Tampermonkey is not opening a tab asking me to log in to Google, like it used to do.

Do you think it is likely to be caused by 1. or 2.?

What should I do to fix it?

Many thanks.

**Comment by derjanb** [comment]
A short test showed Google Drive sync working as expected here with 78b3.

> "Include Tampermonkey settings"

This acts only on the "Utilities" tab. So the shouldn't be able cause your issue.

Can you please create a new temporary profile and double check whether it is working there? All you need to do is to create the profile, install TM and enable sync. Thanks.

**Comment by joeytwiddle** [comment]
Thanks derjanb, I was indeed confused by the Utilities tab. I guess that's just for exported data?

For some reason my sync was disabled, but it worked fine when re-enabling.

(With the exception that it didn't sync which scripts I had turned off in the other browser. They all came down enabled, and I had to turn a few off manually. But I can see that makes sense for people who want to enable/disable different scripts in different browsers.)
